agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -380,6 +380,7 @@ bool backhaul_manager::finalize_slaves_connect_state(bool fConnected,
         network_utils::iface_info iface_info;
         bool backhaul_manager_exist = false;
 
+        notification->params().prplmesh_controller = pSocket->prplmesh_controller;
```

Comment:
I don't think there's a need to save this flag per socket. We can just use `this->prplmesh_controller` here.